pyengine/katana.py

```diff
@@ -48,8 +52,10 @@ def matcher(side,qty,book):
 
                 ordqty = bqqty # in a normal case we use the full quote
                 if bqqty > qty: # if we dont need the whole thing, we just use what we actually need
-                    print('quote is larger than needed {}, will only use needed qty: {}'.format(qty,bqqty))
+                    print('quote is larger than needed {}, will only use needed qty from available {}'.format(qty,bqqty))
                     ordqty = qty
+            else:
+                unusedquotes = '{} {} ({} @ {}),'.format(unusedquotes,quote,book[quote][1],price)
```

Comment:
doesn't rly bother me but we use "book[quote][1]" more than once so we may as well assign it to a variable and use it instead